packages/gen-ai/frontend/src/app/Chatbot/ChatbotPlayground.tsx

```
@@ -280,32 +383,137 @@ const ChatbotPlayground: React.FC<ChatbotPlaygroundProps> = ({
+      onClosePane?.(configId);
+    },
+    [onClosePane],
+  );
+
+  // Shared message bar for both single and compare modes
+  const renderMessageBar = () => (
+    <MessageBar
+      onSendMessage={(message) => {
+        if (typeof message === 'string') {
+          // Send to all instances
+          messageHooksRef.current.forEach((hook) => hook.handleMessageSend(message));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)